Documentation/deploy-master.md

```diff
@@ -6,12 +6,9 @@ See the [CoreOS Documentation](https://coreos.com/os/docs/latest/) for guides on
 
 Manual configuration of the required master node services is explained below, but most of the configuration could also be done with cloud-config, aside from placing the TLS assets on disk. For security reasons, these secrets should not be stored in cloud-config.
 
-The instructions below configure the required master node services using two main directories:
+The instructions below configure the required master node services using:
 
-* `/etc/kubernetes/manifests` for services that the kubelet should run on every master node
-* `/srv/kubernetes/manifests` for services that should run only on a single master node at a time (determined by a leader election).  The podmaster on that node manages the startup of these services by copying their manifests from here to the `/etc/kubernetes/manifests` directory, and the kubelet picks them up from there.
-
-It is vital to understand the distinction between the two and ensure that the scheduler and controller manager manifests are not accidentally placed directly in the `/etc/kubernetes/manifests` directory.
+* `/etc/kubernetes/manifests` for services that the kubelet should run on every master node. The kubelet will pick up services placed here.
```

Comment:
now that this is a single item list, I think can probably collapse it into the sentence above:

> The instructions below configure the required master node services using manifests stored in `/etc/kubernetes/manifests`. The kubelet will watch this location for new or modified manifests and run them automatically.